src/store/fileUploadStore.js

```diff
@@ -179,6 +190,11 @@ const mutations = {
 		state.currentUploadId = currentUploadId
 	},
 
+	// Set prompt and suffix in case of successful propfind attempt
+	setKnownPathSuffix(state, { promptPath, suffix }) {
+		Vue.set(state.knownPaths, promptPath, suffix)
+	},
```

Comment:
Path suffix seams to be some intermediate value to be used only in uploading loop. It doesn't seem to me to be a part of the global reactive application state. Do we need it in the store?

Reply:
For example, it would break any further possibility to perform parallel uploads of different duplicates.

Reply:
> it would break any further possibility to perform parallel uploads of different duplicates.

Then we could scope it within only this store action as a local const? Although it has been done to reuse value between uploads (for example, sharing one file twice with different people make 1..21 requests first time and only 22 second time), I see it as a rare use case